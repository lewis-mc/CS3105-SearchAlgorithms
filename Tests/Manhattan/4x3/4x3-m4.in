4 3 1 2 3 4 5 6 8 0 9 7 10 11