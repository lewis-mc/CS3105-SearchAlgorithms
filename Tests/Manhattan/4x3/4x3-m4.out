1 4 8 7 10 11